tests/Dom_2/dom.js

```diff
@@ -0,0 +1,53 @@
+const file_path = process.env.FILE_PATH;
+const webdriver = require('selenium-webdriver');
+const assert = require('chai').assert;
+const { By, Actions } = require("selenium-webdriver");
+
+
+describe("Testing Scholars Project", function () {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder()
+
+        .forBrowser('chrome')
+        .build();
+
+    var actions = driver.actions();
+
+    before(function setupWebdriver(done) {
+        var file_url = `file://${file_path}`;
+        driver.get(file_url).then(done);
+    });
+
+    after(function () {
+        driver.quit();
+    });
+
+    it('test case: check dot on Cart', async () => {
+        const click = await driver.findElements(By.css("button"));
+        click[2].click();
+        const option = await driver.findElement(By.className("badge")).isDisplayed();
```

Comment:
I don't think you want to search by class because you don't know what the class name the scholar will have set on their own end.

Reply:
I notice you used `findElement` here unlike the others that is `findElements`. Can you check again that everything is fine?